stdlib/websub/src/main/ballerina/src/websub/hub_persistence.bal

```diff
@@ -17,38 +17,38 @@
 # Represents the hub persistence configuration and functions.
 public type HubPersistenceStore abstract object {
 
-    # Function to add or update subscription details.
+    # Adds or updates subscription details.
     #
     # + subscriptionDetails - The details of the subscription to add or update
-    # + return - `error` if an error occurred while adding the subscription, `()` otherwise
+    # + return - An `error` if an error occurred while adding the subscription, `()` otherwise
     public function addSubscription(SubscriptionDetails subscriptionDetails) returns error?;
 
-    # Function to remove subscription details.
+    # Removes subscription details.
     #
     # + subscriptionDetails - The details of the subscription to remove
-    # + return - `error` if an error occurred while removing the subscription, `()` otherwise
+    # + return - An `error` if an error occurred while removing the subscription, `()` otherwise
```

Comment:
```suggestion
    # + return - An `error` if an error occurred while removing the subscription or else `()` otherwise
```
Need to specify the error type.